tools/firtool/firtool.cpp

```diff
@@ -110,9 +110,15 @@ static cl::opt<bool>
                            cl::desc("Convert all chisel asserts into SVA"),
                            cl::init(false), cl::cat(mainCategory));
 
-static cl::opt<bool> stripMuxPragmas("strip-mux-pragmas",
-                                     cl::desc("Don't emit mux pragmas"),
-                                     cl::init(false), cl::cat(mainCategory));
+static cl::opt<bool> addMuxPragmas("add-mux-pragmas",
+                                   cl::desc("Annotate mux pragmas"),
+                                   cl::init(false), cl::cat(mainCategory));
+
+static cl::opt<bool> stripMuxPragmas(
```

Comment:
Thanks for considering the firtool deployment/migration pain points!

Would've been nice if these could be alias options ( https://llvm.org/docs/CommandLine.html#argument-aliases ) but out-of-the-box that doesn't seem possible since they have opposite values.

Consider marking the deprecated option 'hidden', although indicating that in the description gets most of the value there :+1: .

Just curious, is this something expected to be removed in a release or two?

Reply:
> Just curious, is this something expected to be removed in a release or two?

Yes, I'm planning to remove the option by sifive 1.28.